### PR TITLE
CALS-5812: shiro yaml config

### PR DESCRIPTION
**Pull request by DmitriyRudenko**
### JIRA Issue Link
- https://osi-cwds.atlassian.net/browse/CALS-5812

### Tests
- [ ] I have included unit tests 
- [ ] I have included integration tests 
- [ ] I have included performance tests 
- [ ] I have included other tests 
- [x] I have NOT included tests 

### Types of changes

- [ ] New feature (non-breaking change which adds functionality)
- [ ] Refactoring (No behavioral changes)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### CODE REVIEW

**Comment by DmitriyRudenko** [comment]
@LeonidMarushevskyi , too fast, task still is in development :-)

**Comment by DmitriyRudenko** [comment]
retest this please

**Comment by DmitriyRudenko** [comment]
docs: https://github.com/ca-cwds/api-core/wiki/Shiro-configuration-in-YML

File: config/cals-api.yml [hunk 1]
```diff
@@ -20,7 +20,9 @@ server:
       - type: console
 shiro:
   iniConfigs: ["file:${SHIRO_CONFIG_PATH:-config/shiro.ini}"]
-
+  ini:
+    main:
+      perryRealm.validationUri: ${PERRY_VALIDATION_URL:-http://localhost:18080/perry/authn/validate}
```
Comment (vladimirPe) [inline config/cals-api.yml]: Please add description for PERRY_VALIDATION_URL in the README.md file

**Comment by DmitriyRudenko** [comment]
task for PERRY_VALIDATION_URL addition : https://osi-cwds.atlassian.net/browse/DOE-2988

File: config/cals-api.yml (hunk 1 above)
Comment (DmitriyRudenko) [reply config/cals-api.yml]: done

**Comment by DmitriyRudenko** [comment]
api-core version is updated

**Comment by LeonidMarushevskyi** [comment]
Jenkins retest this please

**Pull request merged**